src/server.c

```diff
@@ -1524,7 +1527,7 @@ void afterSleep(struct aeEventLoop *eventLoop) {
             latencyStartMonitor(latency);
 
             moduleAcquireGIL();
-
+            moduleFireServerEvent(REDISMODULE_EVENT_AFTER_SLEEP, 0, NULL);
```

Comment:
Maybe check `moduleCount()` to avoid a needless call, like `beforeSleep`?

Reply:
This is inside `moduleCount()` check already, please see line 1525.

Reply:
Sure, my bad!